src/elements/common/base.scss

```diff
@@ -89,6 +83,20 @@
     }
 }
 
+// Elements migrated to Blueprint should not use base styles
+.bcow,
+.bcp,
+.bcpr,
+.bcs,
+.bdl-UnifiedShareForm {
```

Comment:
these classes exist at the same level as the `.be` class on the elements. so this has the same specificity as the block above